doc/rbd/index.rst

```diff
@@ -4,18 +4,19 @@
 
 .. index:: Ceph Block Device; introduction
 
-A block is a sequence of bytes (often 512).
-Block-based storage interfaces are a mature and common way to store data on
-media including HDDs, SSDs, CDs, floppy disks, and even tape.
-The ubiquity of block device interfaces is a perfect fit for interacting
-with mass data storage including Ceph.
-
-Ceph block devices are thin-provisioned, resizable, and store data striped over
-multiple OSDs.  Ceph block devices leverage
+RBD (the Ceph block device) is a service that provides thin-provisioned,
```

Comment:
I _think) it's possible to thick-provision, but it's rarely done, so probably no need to complicate here.

Reply:
Yes, it's possible but it's a niche use case.  At the same time we definitely want to highlight thin-provisioning because a lot of properties fall out of that, so +1 to not complicating here.